pelican/Dockerfile

```diff
@@ -50,7 +50,7 @@ COPY requirements.txt .
 RUN pip install -r requirements.txt --no-deps
 
 # Could perhaps be added to requirements.txt but that would affect other uses
-RUN pip install 'MarkupSafe<2.1.0' # needed for Pelican 4.5.4
+RUN pip install 'MarkupSafe==2.0.1' # needed for Pelican 4.5.4
```

Comment:
Idea: Maybe a good idea would be to add comment here to link to the issue/problem to solve it. Usually when I am doing such exclusion I create an issue that describes the "permanent" solution even if for now we want to temporarily limit it. Pinning a dependency inside dockerfile is quite problematic because almost for sure we will forget to unpin it or migrate (and we have now dependabot added to the repo to bump dependencies automatically, so likely some solution where dependabot will be able to update pinning automatically would be better.

Reply:
100% agree